README.rst

```diff
@@ -4,18 +4,45 @@ Oculy: Modular data viewer
 Installation
 ------------
 
-Currently Oculy cannot be installed directly due to one of its dependency not
-being available on PyPI. To install it directly from GitHub, you can::
+First, all you need is to clone the repository of oculy on your local machine::
 
-    pip install git+https://github.com/MatthieuDartiailh/glaze
-    pip install git+https://github.com/MatthieuDartiailh/oculy
+    git clone https://github.com/MatthieuDartiailh/oculy.git
 
-Once installed Oculy can be started by either running::
+Now that you cloned oculy, you are to to build your package. You may need to install the build
+package first::
+
+    pip install build
+
+OThis should create a `build/` folder as well as a `dist/` folder
+(don't forget to add these to your `.gitignore`, if they aren't in there already.
```

Comment:
Irrelevant, the gitignore belongs to the project so we should add build to it ourselves.

Reply:
Yes. This is exactlry what I meant. The folders build/ and dist/ have to be added by ourselves.

Reply:
So please do it as part of this PR

Reply:
> So please do it as part of this PR

These foldres are add by IDE PyCharm but they will not be commited onto the repository so that I could pull a request to the main repo.

Reply:
the dist and build folder are created by Python when building the package and are actually already listed under gitignore https://github.com/MatthieuDartiailh/oculy/blob/main/.gitignore#L12 and L14